Formula/cfitsio.rb

```diff
@@ -39,4 +40,18 @@ def install
       end
     end
   end
+
+  test do
+    cp Dir["#{pkgshare}/testprog/testprog*"], testpath
+    flags = (ENV.cflags || "").split + (ENV.cppflags || "").split + (ENV.ldflags || "").split
```

Comment:
So I removed this line. What about a mass removal from the other 60+ formulae to prevent its further spread?